ENH: improve order of tasks (#152)
TheAxonLab/hcph-sops: docs/data-collection/tear-down.md

    At the same time, you carefully feed them so they do not suffer abrasion from touching the edges of the tube.

    !!! warning "Extracting the cables and tubes requires two experimenters"

- [ ] The experimenter currently outside in the Control Room will re-enter the Scanner Room with the plastic bag of the mirror and a fresh pair of gloves and will leave them prepared on the bed.
- [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.

    !!! danger "The infrared mirror MUST be manipulated with clean gloves at all times."

- [ ] Immediately insert the infrared mirror back into its plastic pouch.
- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
- [ ] Re-attach the mirror to the coil's frame.
- [ ] Take the projector's screen off and store it in its designated shelf.
- [ ] Disconnect the ECG cable from the filter of the access panel, roll the cable.

> ### Reading the scanner for the next session

Author: ```suggestion
### Readying the scanner for the next session
```

- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC.
- [ ] Carefully roll the GA tube and put it in its designated bag.
- [ ] Carefully roll the RB tube and put it in its designated bag.
- [ ] Disconnect the MMBT-S Interface, and the two corresponding cables (pink USB, 25-pin parallel), and insert them in their designated bags.
- [ ] Remove the cable connecting the GA to the AMI100D module of the BIOPAC, roll it and put it in its designated bag.
- [ ] Put the GA and the corresponding cables in the designated box.
- [ ] Disconnect and bag the remaining cables connected to the BIOPAC:
    - [ ] Ethernet to the *{{ secrets.hosts.acqknowledge | default("███") }}* computer.
    - [ ] Power cord.
- [ ] Retrieve and bag the *AcqKnowledge USB License Key* from the *{{ secrets.hosts.acqknowledge | default("███") }}* computer.
- [ ] Store the *fMRI box*, BIOPAC and GA boxes back in room {{ secrets.rooms.et_camera| default("███") }}.

### Finalizing with the ET's PC

- [ ] **Retrieve ET recordings** (from {{ secrets.hosts.psychopy | default("███") }}):
    - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}*

>     - [ ] Upload to a pre-designated drop-box (e.g., using Dropbox)

Reviewer: @esavary I haven't been able to interpret this. Is this what we are doing? What did AcqKnowlege have to do with this?

Once this item is clear, we are ready to merge this one.

Author: I removed lines 113-115 since they were no longer relevant. I am considering moving the transfer to Dropbox under "data management".WDYT?

Reviewer: I'd be more confident if we make sure the files are uploaded to Dropbox on the spot.

We should change the psychopy experiments to write directly on the shared folder instead of in the git repo.

Reviewer: See TheAxonLab/HCPh-fMRI-tasks#24

## Showing the participant out

- [ ] Extract the bed from the scanner's bore using the scanner's control wheel.

>     The home (:fontawesome-solid-house:) button can be used to bring the bed out.

Reviewer: 👏

>     The home (:fontawesome-solid-house:) button can be used to bring the bed out.

Reviewer 2: ```suggestion
    The home (:fontawesome-solid-house:) button can alternatively be used to bring the bed out.
```

>     The home (:fontawesome-solid-house:) button can alternatively be used to bring the bed out.

Reviewer: Several things are repated or suggested later, but should be here:

After line 22
```
- [ ] Prepare the belt on the bed to be removed from the room when you show the participant out.
```

After line 23 (after adding the above line, 24)
```
- [ ] Disconnect the last section of the cannula and dispose of it in the trash can.
```

- [ ] Give the participant the corresponding compensation for the participation and transportation.
- [ ] Request a signed receipt of the amount of the financial compensation.

!!! important "To boost efficiency, two people can work simultaneously on the following steps — one inside the scanning room and the other outside. They can follow the designated sections [inside the scanning room](#after-scan-inside-the-scanner-room) and [outside the scanning room](#after-scan-outside-the-scanner-room) in parallel."

## AFTER SCAN, inside the scanner room

- [ ] Unplug the two cables connected to the ET (signal and power). Put those extremities aside far from the scanner.

> - [ ] Disconnect the GA and RB tubes.

Reviewer: This is repeated, done already in ~22-24
```suggestion
```

> - [ ] **With someone outside in the control room**:
>     - [ ] Carefully extract the four cables one by one (the fiber and power of the ET, the tube of RB and the tube of the GA) back through the access tube. 
>   

Reviewer: This needs more coordination (like for the communication to check the tightness of the RB)
```suggestion
- [ ] Agree with another experimenter, who will exit the room, sit on the other side of the access tubes in the Control Room, and pull cables (fiber and power of the ET) and tubes (RB and GA) out of the room.
    At the same time, you carefully feed them so they do not suffer abrasion from touching the edges of the tube.
```

> - [ ] **Carefully remove the infrared mirror**:
>     - [ ] Enter again the scanner room with the plastic container of the mirror and leave it prepared on the bed.
>     - [ ] Separate the mirror frame from the upper part of the head coil and lay it on the bed.

Reviewer: This comment also replaces the following four lines (36-39 of the original text), GitHub does not allow me to modify them directly.

```suggestion
- [ ] The experimenter currently outside in the Control Room will enter the Scanner Room with the plastic bag of the mirror and a fresh pair of gloves and will leave them prepared on the bed.
- [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.
- [ ] Immediately insert the infrared mirror back into its plastic pouch.
- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
- [ ] Re-attach the mirror to the coil's frame.
```

- [ ] Give the participant the corresponding compensation for the participation and transportation.
- [ ] Request a signed receipt of the amount of the financial compensation.

> !!! important "To boost efficiency, two people can work simultaneously on the following steps — one inside the scanning room and the other outside. They can follow the designated sections [inside the scanning room](#after-scan-inside-the-scanner-room) and [outside the scanning room](#after-scan-outside-the-scanner-room) in parallel."

Reviewer: Too long for a title, let's use the full admonition:
```suggestion
!!! important "The following steps are better done with two people"
    One researcher operates **inside** the Scanning Room and the other one is **outside**, at the Control Room.
    They will parallelize the designated steps below: [inside the scanning room](#after-scan-inside-the-scanner-room), and [outside the scanning room](#after-scan-outside-the-scanner-room).
```

## AFTER SCAN, inside the scanner room

> - [ ] Unplug the two cables connected to the ET (signal and power). Put those extremities aside far from the scanner.

Reviewer: ```suggestion
- [ ] Unplug the two cables (signal and power) connected to the ET arm.
    Put those extremities aside far from the scanner.
```

- [ ] Disconnect the GA and RB tubes.

> 

Reviewer: ```suggestion
```

- [ ] **With someone outside in the control room**:

>     - [ ] Carefully extract the cables (fiber and power of the ET) back through the access tube. 

Reviewer 2: ```suggestion
    - [ ] Carefully extract the four cables one by one (the fiber and power of the ET, the tube of RB and the tube of the GA) back through the access tube. 
```

>     - [ ] Extract the RB and the GA tubes from the room.

Reviewer 2: Is there a reason why the RB and GA tubes as distinguished from the ET cables? We used to remove everything at the same time through the same hole, so might as well get concatenate in one sentence.
```suggestion
```

- [ ] Cleaning up instrumentation:
    - [ ] Take the projector's screen off and store it in its designated shelf.

>     - [ ] Take the ET outside and put it in a stable place.
>     - [ ] Disconnect the ECG leads from the filter of the access panel, fold the cable and leave it prepared with the RB to take out of the room with other equipment.
>     - [ ] Disconnect the last section of the cannula and dispose of it in the trash can.

Reviewer: The cannula tube has already been removed at this point
```suggestion
- [ ] Disconnect the ECG cable from the filter of the access panel, roll the cable.
- [ ] Take the ET arm and the ECG cable outside to the Control Room.
- [ ] Place the ET arm in a stable place.
- [ ] Store the ECG cable in its designated box / storage.
- [ ] Store the ET arm in its designated box.
```

    - [ ] Remove the back padding elements and put them back in their designated storage.
    - [ ] Reinstall the spine coil.
    - [ ] Plug back the head coil if you know the next exam will require that specific coil, or simply put it away with the other (head) coils on the shelf next to the scanner.
    - [ ] Put the bed back in place = push the home (:fontawesome-solid-house:) button on the scanner

>     - [ ] Put the wooden stopper of the main access tube back on the tube.

Reviewer 2: Because the GA position has changed, not relevant anymore
```suggestion
```

    - [ ] Everything that is removed for the experiment MUST be put back in place at the end of the experiment, i.e. position of the bed, coil, emergency button, ears padding.
    - [ ] Take a glove, on the right there is some cleaning napkins. Use them to clean the bed.
    - [ ] Take the ECG electrodes, the RB, and the plexiglas outside to the control room.
    - [ ] Close the external door.

## AFTER SCAN, outside the scanner room

- [ ] Help the person inside the scanning room to extract the cables and tubes.
- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC Tower.
- [ ] Carefully roll RB and the GA tubes and put them back in the GA and BIOPAC boxes.

> - [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its cover.

Reviewer 3: Below (original line 88), there is an instruction to "Put back the regular Lens" - should it be added here as well ?

Author: I think we actually stopped putting the original lens back on the ET, possibly to ease the storage. Maybe @celprov or @ bfranceschiello can confirm this?

Author: I just had a confirmation from @ bfranceschiello , we do not need to put it back.

Reviewer 3: Oh alright, my bad, thanks for the clarification :)

  ![cover-mri-compatible-lens](../assets/images/cover-mri-compatible-lens.png "Cover MRI compatible lens")

- [ ] Put the cover, the ET base back in the fMRI box, **being extremely careful to not crush the mirror**.
- [ ] **Retrieve ET recordings** (from {{ secrets.hosts.psychopy | default("███") }}):
    - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}* and save the experiment from AcqKnowledge.
    - [ ] Upload to a pre-designated drop-box (e.g., using Dropbox)

> - [ ] Press <span class="keypress">Ctrl</span>+<span class="keypress">Alt</span>+<span class="keypress">Q</span> on the ET PC Tower to exit the EyeLink 1000 Plus Host PC application and click on the <span class="keypress">Shutdown</span> button from the **File Manager** toolbar.

Reviewer 2: @esavary Have you tested this on our setup? From what I remember, when I exited the Eyelink application, I was left with a black screen, so "click on the Shutdown button from the File Manager toolbar" would not have been possible.

Author: I will double-check the EyeTracker. The last time I tried to exit the application, I had a DOS-like screen.

Reviewer: Since this comment has not been resolved within this PR, I just opened #166 to follow up.

> - [ ] Switch off laptop and ET PC Tower. Plug back the sync box and the VGA projector where they were.

Reviewer 2: ```suggestion
- [ ] Switch off the laptop. Plug back the sync box and the VGA projector where they were.
```

- [ ] Fix the rolled cable with the scotch on the ET PC Tower.
- [ ] Turn off the pump of the GA, then switch the GA off. **DO NOT PUT THE CAP IN WHILE THE PUMP IS ON.**
- [ ] Remove the cables connected to the BIOPAC and the GA and store them in the boxes in their original bags.
- [ ] Bring back the fMRI, BIOPAC and GA boxes in {{ secrets.rooms.et_camera| default("███") }}.

> - [ ] Bring back the ET PC Tower and the plexiglas in {{ secrets.rooms.projector | default("███") }}

Reviewer 2: ```suggestion
- [ ] Bring back the ET PC Tower and the plexiglas in {{ secrets.rooms.projector | default("███") }}.
```
